packages/wallets/src/features/cashier/modules/TransactionStatus/utils/transactionDisplayMapper.ts

```diff
@@ -0,0 +1,130 @@
+import moment from 'moment';
+import { getTruncatedString } from '@deriv/utils';
+import { THooks } from '../../../../../types';
+
+const transactionDetailsMapper = (transaction: THooks.CryptoTransactions) => {
+    const transactionHashObscure = transaction.transaction_hash
+        ? `${getTruncatedString(transaction.transaction_hash, { length: 8, type: 'middle' })}`
+        : 'Pending';
+
+    if (transaction.transaction_type === 'deposit') {
+        return {
+            CONFIRMED: {
+                confirmationDisplay: 'Confirmed',
```

Comment:
that feels like translation concern - wondering if there is a way to push this to translations system, so we wouldn't need any mapping for translations itself,

Reply:
In the same task but for FB2 (https://github.com/binary-com/deriv-app/pull/10469/) the strings were wrapped in `localize`

Reply:
https://github.com/rostislav-deriv/deriv-app/blob/29a79ef10a04592e90f2b9e523aef12f2cac576c/packages/cashier/src/modules/crypto-transactions/helpers/crypto-transactions-mapper.tsx

Reply:
hmm, I was thinking not about putting translations to mapping, but putting mapping to translations and thus completely remove the need for mapping file (so no extra hoops). Basically instead of going through mapper, go directly ;)

Reply:
You mean translating directly from this object's keys the same way as with `statusColor`?